pkg/volume/iscsi/iscsi_util.go

```diff
@@ -51,6 +52,9 @@ const (
 
 	// How many seconds to wait for a multipath device if at least two paths are available.
 	multipathDeviceTimeout = 10
+
+	// 'iscsiadm' error code stating that a session is logged in
+	iscsiadmErrorSessExists = 15
```

Comment:
For reviewers, iscsiadm use code per error:

https://github.com/open-iscsi/open-iscsi/blob/7d121d12ad6ba7783308c25ffd338a9fa0cc402b/include/iscsi_err.h#L37-L38

Reply:
thanks @redbaron 
@bertinatto please add this reference in your comment